index.js

```diff
@@ -713,6 +713,12 @@
   //. Type comprising every primitive String value.
   var String_ = NullaryTypeWithUrl('String', typeofEq('string'));
 
+
+  //# Symbol :: Type
+  //.
+  //. Type comprising every primitive Symbol value.
+  var Symbol_ = NullaryTypeWithUrl('Symbol', typeofEq('symbol'));
```

Comment:
They should be equivalent. I would think the `typeofEq` would be slightly faster. I don't know if there are any benchmarks though.